### ISSUE TITLE
Accessibility (Easy) - Text Alternatives (Guideline 1.1.1)

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/lsf#141]
**[Original report](https://bitbucket.org/laborstudents/lsf-flask/issue/141) by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

Text alternatives are needed for all non-text content. This would include the Labor Office logo. I don’t think it would include anything else on the interface.

‌

Example using the Labor Office logo:

```
<img src="/static/laborprogram.png" alt=“Berea Labor Program Logo”/>
```

‌

If you need more information about this guideline, please visit the wiki \([http://172.31.2.178/accessibility/checklist](http://172.31.2.178/accessibility/checklist)\) or ask Hailey.

**Comment by bryantal** [comment]
**Original comment by Shageldi Ovezov (Bitbucket: [Shageldi Ovezov](https://bitbucket.org/Shageldi Ovezov), ).**

----------------------------------------

Started working on this

**Comment by bryantal** [comment]
**Original comment by Shageldi Ovezov (Bitbucket: [Shageldi Ovezov](https://bitbucket.org/Shageldi Ovezov), ).**

----------------------------------------

So the only place in the whole application where I could find a tag that accepts the alt attribute was the logo image. I did some research and found out that alt can also be used with the inputs that are type of image. Since we don’t have such inputs, I am not sure where else we could include alternative text. I also searched if I could add alternative text for glyph icons but wasn’t successful in finding a feature that does so. Any suggestions @{5cf5bb8823e75a0e7d27d072} , @{5cdab02ca649ce0dcd8101f0} ?

**Comment by bryantal** [comment]
**Original comment by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

I am pretty sure that the only place that alt text is needed is the logo. If that was implemented, I would consider this complete.

**Comment by bryantal** [comment]
**Original comment by Brian Ramsay (Bitbucket: [BrianRamsay](https://bitbucket.org/BrianRamsay), GitHub: [BrianRamsay](https://github.com/BrianRamsay)).**

----------------------------------------

Merged in issue141 (pull request #118)

Fixed issue 141

Approved-by: Brian Ramsay
